contribs/analysis/src/main/java/org/matsim/contrib/analysis/vsp/traveltimedistance/HereMapsRouteValidator.java

```diff
@@ -126,9 +128,13 @@ public HereMapsRouteValidator(String outputFolder, String apiAccessKey, String d
                     bw.close();
                 }
             }
+
+            in.close();
         } catch (MalformedURLException e) {
-        } catch (IOException e) {
-        } catch (ParseException e) {
+            log.error("URL is not working. Please check your API key");
+            e.printStackTrace();
```

Comment:
printStackTrace can be removed and instead exception passed to logger:
log.error("URL is not working. Please check your API key", e);